src/api/todos.ts

```diff
@@ -0,0 +1,15 @@
+import { Todo } from '../types/Todo';
+import { NewTodo } from '../types/NewTodo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = (someTodo: NewTodo) => {
```

Comment:
Consider using a more descriptive parameter name instead of 'someTodo'. For example, 'newTodo'.

Reply:
done